readthedocs/templates/projects/domain_form.html

```diff
@@ -36,6 +36,15 @@
       {% endblocktrans %}
     </p>
 
+    {% if not domain.skip_validation and not domain.is_valid and domain.validation_process_expired %}
+      <p class="help_text">
+        {% blocktrans trimmed %}
+        The validation process period has ended.
+        Save the domain to restart the process.
```

Comment:
:+1: for now, but I suppose we could come back to this and alter the button too. Like, if the validation period is over, change the button text to "Retry validation" or something similar